hyperspy/_signals/eds_tem.py

```diff
@@ -562,6 +689,36 @@ def create_model(self, auto_background=True, auto_add_lines=True,
                             *args, **kwargs)
         return model
 
+
+    def _get_probe_area(self, navigation_axes=None):
+        """Calculates a pixel area which can be approximated to probe area,
+        when the beam is larger than or equal to pixel size.
+        """
+        if (self.axes_manager.navigation_dimension > 2 and
+                navigation_axes is None):
+            raise ValueError("With `probe_area='auto' and "
+                             "navigation dimension > 2, you need "
```

Comment:
Change to: "The navigation axes corresponding of the probe are ambiguous, you need to specify the `navigation_axes` parameter."?

Even in the case of `navigation_dim == 2`, it assumes that these two navigation axes correspond to these of the probe, which will be wrong in case of stack of line scan, as in case of a time serie of line scan, for example.

Reply:
Fair point I hadn't thought of that. Is there a way around this or just a question of documenting it?

Reply:
I have tried to fix this in https://github.com/k8macarthur/hyperspy/pull/17.